src/tree/updater_gpu_hist.cu

```diff
@@ -1446,7 +1446,9 @@ class GPUHistMakerSpecialised{
 
     // Find the cuts.
     monitor_.StartCuda("Quantiles");
-    common::DeviceSketch(batch, *info_, param_, &hmat_, hist_maker_param_.gpu_batch_nrows);
+    // TODO(sriramch): The return value will be used when we add support for histogram
+    // index creation for multiple batches
+    (void)common::DeviceSketch(param_, hist_maker_param_.gpu_batch_nrows, dmat, &hmat_);
```

Comment:
No need to cast the return value to `void`. Just ignoring it is fine.

Reply:
i'll clean all these up in a follow-up pr for the histogram. i added a todo and did this just to remind myself of this. the void return casting is to just to notate for now that the return value is ignored

Reply:
A TODO describing that you ignore the value is fine. No need to cast to `void`, ignoring return values in C/C++ is OK.